sdks/python/apache_beam/transforms/util.py

```diff
@@ -780,6 +783,48 @@ def expand(self, pcoll):
             self.max_buffering_duration_secs,
             self.clock))
 
+  @typehints.with_input_types(Tuple[K, V])
+  @typehints.with_output_types(Tuple[K, Iterable[V]])
+  class WithShardedKey(PTransform):
```

Comment:
Should this be tagged experimental similar to the Java change?

Reply:
Done.